uv.lock

```diff
@@ -1,4 +1,5 @@
 version = 1
+revision = 2
```

Comment:
The lock file has a lot of churn alongside the package removals because it's also getting updated for the latest 0.7.x series, which now includes `upload-time` in lock entries: https://github.com/astral-sh/uv/releases/tag/0.7.0

I can split this churn out into a different PR if you'd like.